egs/csj/ASR/local/compute_fbank_csj.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+# Copyright    2022  The University of Electro-Communications  (Author: Teo Wen Shen)  # noqa
+#
+# See ../../../../LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import logging
+import os
+from itertools import islice
+from pathlib import Path
+from random import Random
+from typing import List, Tuple
+
+import torch
+from lhotse import (
+    ChunkedLilcomHdf5Writer,
```

Comment:
Could you switch to `LilcomChunkyWriter` like the following one in librispeech?
https://github.com/k2-fsa/icefall/blob/d1f16a04bd2de8f0203eb0a43051c2ec3e91fb11/egs/librispeech/ASR/local/compute_fbank_librispeech.py#L31

Reply:
Yes sure. Is there a reason why LilcomChunkyWriter is preferred? 

I was thinking of including that in a comment.

Reply:
From https://github.com/k2-fsa/icefall/pull/404#issuecomment-1148650517

> HDF5 unfortunately will cause issues to ‚big data’ users…

Reply:
Also, please see https://github.com/lhotse-speech/lhotse/pull/527

---

`LilcomChunkyWriter` does not depend on hdf5 -> one fewer dependency.